eng/pipelines/official.yml

```diff
@@ -1,199 +1,114 @@
-# Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+# Pipeline: https://dev.azure.com/dnceng/internal/_build?definitionId=1298
 
-# Name: dotnet-workload-versions-official-ci
-# URL: https://dev.azure.com/dnceng/internal/_build?definitionId=1298
+trigger:
+  batch: true
+  branches:
+    include:
+    - main
+    - release/*
 
-# Produces the signed NuGet packages for workload versioning.
-
-###################################################################################################################################################################
-# PIPELINE METADATA
-###################################################################################################################################################################
-
-# Activates the pipeline after a PR is merged.
-# See: https://docs.microsoft.com/azure/devops/pipelines/yaml-schema/trigger?view=azure-pipelines
-trigger: none
-
-# Activates PR builds for this pipeline.
-# See: https://docs.microsoft.com/azure/devops/pipelines/yaml-schema/pr?view=azure-pipelines#examples
 pr: none
 
 parameters:
-# When true, tries to publish
-- name: StabilizePackageVersion
-  displayName: Stabilize Package Version
+- name: stabilizePackageVersion
+  displayName: Stabilize package version
   type: boolean
   default: false
 - name: publishToFeed
   displayName: Publish to feed
   type: boolean
   default: false
-- name: feedToPublishTo
-  displayName: Feed to publish to
+- name: feedForPublishing
+  displayName: Feed for publishing
+  type: string
+  default: public/dotnet9-workloads
+- name: createVSInsertion
+  displayName: Create VS insertion
+  type: boolean
+  default: false
+- name: usePreComponentsForVSInsertion
+  displayName: Use Preview Components for VS insertion
+  type: boolean
+  default: false
+- name: vsTopicBranch
+  displayName: 'VS Topic Branch [default: temp/{team}/{target}/yyyy-MM]'
   type: string
-  default: 'public/dotnet9-workloads'
+  default: '|default|'
+- name: workloadDropNames
+  displayName: Workload drop names
+  type: object
+  default:
+  - emsdk
+  - mono
```

Comment:
Did you ever end up getting a drop of maui, android, or ios that worked for this testing?

Reply:
Only one I got to test was an `android` package that was made specifically for testing. So, until they make the actual changes necessary (drop name formatting, .metadata file creation, update VS components json, and publish to the correct channel), then they can add their name here.

Reply:
I'm not sure from our meeting with them that they are tracking and planning to do this work. Can you ping them offline and we can bring it up Monday?